examples/etcd/etcd-in-cluster/README.md

```diff
@@ -0,0 +1,127 @@
+This document is going to show you how you can backup the ETCD cluster that is running as part of your Kubernetes control plane. The
+commands are run into a cluster that is setup using [Kind](https://kind.sigs.k8s.io/docs/user/quick-start/) but it should work on any other single or multi node ETCD cluster.
+
+## Prerequisites Details
+
+* Kubernetes 1.9+ with Beta APIs enabled, and you are not on managed Kubernets
+* PV support on the underlying infrastructure.
+* Kanister version 0.32.0 with `profiles.cr.kanister.io` CRD installed
+
+# Integrating with Kanister
+
+Once we have made sure that the prequisites are met, when we say integrating with Kanister we mean creating some CRs, for example Blurprint and Actionset
+that would help perform the actions on the the ETCD instance that we are running.
+
+## Create Profile resource
+
+```bash
+» kanctl create profile s3compliant --access-key <aws-access-key> \
+        --secret-key <aws-secret-key> \
+        --bucket <bucket-name> --region ap-south-1 \
+        --namespace kanister
+secret 's3-secret-7umv91' created
+profile 's3-profile-nnvmm' created
+```
+This command creates a profile which we will use later.
+
+The command will configure a location where artifacts resulting from Kanister
+data operations such as backup should go. This is stored as a `profiles.cr.kanister.io`
+*CustomResource (CR)* which is then referenced in Kanister ActionSets. Every ActionSet
+requires a Profile reference to complete the action. This CR (`profiles.cr.kanister.io`)
+can be shared between Kanister-enabled application instances.
+
+## Create Blueprint
+
+If you are running ETCD on production, there would be some authentication mechanism that your cluster is using. Since we are taking an example
+of the cluster that is setup through Kind ([kubeadm](https://github.com/kubernetes/kubeadm)), I am assuming TLS based authentication is being used.
+
+To specify the location of the CA, certificate and key we will have to create a secret in the same namespace where your ETCD pod is running. This
+secret is going to have the name of the format `etcd-<etcd-pod-namespace>` with these fields
+
+- **cacert** : CA (certificate authority) cert, would usually be `/etc/kubernetes/pki/etcd/ca.crt` on Kind clusters
```

Comment:
I think this is in case of all clusters created with kubeadm

Reply:
not really Rahul. But I will improve the statement here.